controller_manager/controller_manager/__init__.py

```diff
@@ -12,8 +12,101 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from . import launch_utils
+from controller_manager_msgs.srv import ConfigureController, \
+    ListControllers, ListControllerTypes, ListHardwareInterfaces, \
+    LoadController, ReloadControllerLibraries, SwitchController, UnloadController
+
+import rclpy
+
+
+def service_caller(node, service_name, service_type, request):
```

Comment:
I know it is common in the ROS codebase to chuck random code in `__init__.py` but I think we should set a better example. Let's create a new file for this with a name indicative of the contents and only use this file to expose them to the outside.